src/utils/proxy.ts

```diff
@@ -31,7 +31,7 @@ export async function proxyRequest(
   opts: ProxyOptions = {}
 ) {
   // Method
-  const method = getMethod(event);
+  const method = opts.fetchOptions?.method ?? getMethod(event);
```

Comment:
Just in case to handle edge case of method: `""`
```suggestion
  const method = opts.fetchOptions?.method || getMethod(event);
```